dts/bindings/display/led-strip-matrix.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Generic LED strip matrix (LED strip arranged in a grid pattern)
+
+compatible: "led-strip-matrix"
+
+include: display-controller.yaml
+
+properties:
+  led-strip:
+    type: phandle
+    required: true
+    description: |
+      Specify the LED strip that is the substance of the matrix.
+
+  circulative:
```

Comment:
I think a more widely used term is "serpentine" for what you call "non circulative"?

Reply:
I couldn't think of a better expression.
It looked good so I applied it.